grails-app/migrations/views/invoice-list.sql

```diff
@@ -1,20 +1,27 @@
 CREATE OR REPLACE VIEW invoice_list AS (
    	SELECT
-         invoice.id,
-		 invoice.party_from_id,
+         invoice.id as id,
```

Comment:
This will break the PK if there's more than one VENDOR_INVOICE_NUMBER reference number, so let's remove this column and use a composite key.